ms-engine-spring-jaxrs/src/main/java/net/trajano/ms/engine/jaxrs/JaxRsRouter.java

```diff
@@ -70,6 +77,8 @@ public void register(final Class<? extends Application> applicationClass,
                     final String[] consumes = Optional.ofNullable(m.getAnnotation(Consumes.class)).map(Consumes::value).orElse(new String[0]);
                     final String[] produces = Optional.ofNullable(m.getAnnotation(Produces.class)).map(Produces::value).orElse(new String[0]);
 
+                    System.out.println(m);
+                    System.out.println(clazz);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)